lib/elasticity/search.rb

```diff
@@ -51,6 +51,12 @@ def documents(document_klass)
         end
       end
 
+      # Performs the search using the default search type and returning an iterator that will yield
+      # each aggregation.
+      def aggregations
+        document_hashes.aggregations
+      end
+
```

Comment:
You shouldn't need this. Also, the comment is wrong since this won't return a iterator. Come over if you want more info about this.